tests/components/mitsubishi_itp/common.yaml

```diff
@@ -21,9 +21,10 @@ uart:
     tx_pin:
       number: GPIO4
 
-mitsubishi_uart:
-  uart_heatpump: hp_uart
-  uart_thermostat: tstat_uart
-  update_interval: 12s
-  temperature_sources:
-    - fake_temp
+climate:
+  - platform: mitsubishi_uart
```

Comment:
Shouldn't this be `mitsubishi_itp`?

Reply:
It should!  Thanks.  (I will definitely make sure I run all the tests before merging to dev, but for some reason the test scripts don't run as written in my dev container and I have to modify some files to get them to run.  Minor inconvenience, but then git keeps wanting to push the changes to the test files, etc., etc.)